install/kubernetes/helm/istio-remote/values.yaml

```diff
@@ -97,6 +90,14 @@ global:
   # Default is 1 second
   refreshInterval: 10s
 
+  # Remote Istio endpoints. Can be hostnames or IP addresses
+  # The Pilot address is required. The others are optional.
+  remotePilotAddress: ""
```

Comment:
Kubernetes applications won't work in this model unless there is some magic I don't understand.  Currently we specify the pilotendpoint so that the sidecars inside kubernetes can find pilot and receive the mesh calculation.

Could you add more detail as to how the deployment works in this situation?

Reply:
I use `remotePilotAddress` as the discoveryAddress [here](https://github.com/istio/istio/pull/6366/files#diff-4e2f8fda81dd49dc86533e3c3af59d8bR106) in the mesh config (along with the other `remoteX` addresses. The proxy sidecars will then use those adresses instead of local default service names.

Reply:
Sounds like an improvement.  Mind I test prior to merge?  cc @baodongli @tiswanso

Reply:
Please!

Reply:
@Rigdon are you using proyv1, or v2?  v1 is deprecated, and IIUC will disappear in the 1.0 release.  We should be marching towards that model if possible.

Reply:
The default in both charts now is the `proxyv2` image, though as @tiswanso pointed out the sidecars are still configured by default to use the old http ports rather than the newer grpc ports. I tried to keep as much the same as possible between the two charts.

Reply:
@sdake -- I think we want to fix this in the main charts ASAP.  From your slack msg, I'm guessing you're looking at it.  Do you know if we have an issue already for tracking the latest xDS config stuff the helm charts?

@baodongli  ^^